src/types.ts

```diff
@@ -23,19 +23,15 @@ export interface Config {
 }
 
 export interface MessageOptions {
-  // sonarignore:start
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  [key: string]: any;
-  // sonarignore:end
+  [key: string]: any; //NOSONAR
 }
```

Comment:
Can use `AnyObject` here too. It serves the same purpose of accepting any object with string as key and value of any type in it.

Reply:
done